src/modules/workspaces/workspaces.usecase.spec.ts

```diff
@@ -949,4 +951,214 @@ describe('WorkspacesUsecases', () => {
       expect(assignableSpace).toBe(BigInt(300000));
     });
   });
+
+  describe('getWorkspaceMembers', () => {
+    const owner = newUser();
+    const workspace = newWorkspace({ owner });
+
+    it('When workspaceId does not exist then it should throw an error', async () => {
+      const workspaceId = 'not-exist-uuid';
+      const findWorkspace = jest.spyOn(workspaceRepository, 'findOne');
+      findWorkspace.mockReturnValueOnce(null);
+
+      expect(async () => {
+        await service.getWorkspaceMembers(workspaceId, owner);
+      }).rejects.toThrow();
+
+      expect(findWorkspace).toBeCalledWith({
+        id: workspaceId,
+        ownerId: owner.uuid,
+      });
+    });
+
+    it('When there are no members then return an empty array', async () => {
+      const workspaceUsers = jest.spyOn(
+        workspaceRepository,
+        'findWorkspaceUsers',
+      );
+      workspaceUsers.mockResolvedValue([]);
+
+      await expect(
+        service.getWorkspaceMembers(workspace.id, owner),
+      ).resolves.toStrictEqual([]);
+
+      expect(workspaceUsers).toHaveBeenCalled();
+    });
+
+    it('When a user is registered as member and is not the owner then the "isOwner" field will be False', async () => {
+      const user = newUser();
+      const workspaceUser = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user.uuid,
+        member: user,
+      });
+
+      const getWorkspace = jest
+        .spyOn(workspaceRepository, 'findOne')
+        .mockResolvedValue(Promise.resolve(workspace));
+
+      const workspaceUsers = jest
+        .spyOn(workspaceRepository, 'findWorkspaceUsers')
+        .mockResolvedValue(Promise.resolve([workspaceUser]));
+
+      const fnGetMembers = service.getWorkspaceMembers(workspace.id, owner);
+      await expect(fnGetMembers).resolves.toMatchObject([{ isOwner: false }]);
+
```

Comment:
Use async await. Also, this does not return the value expected, isn't it? GetWorkspaceMembers always return {activatedUser: [], deactivatedUser: []}

```
 FAIL  src/modules/workspaces/workspaces.usecase.spec.ts (6.179 s)
  ● WorkspacesUsecases › getWorkspaceMembers › When there are no members then return an empty array

    expect(received).resolves.toStrictEqual(expected) // deep equality

    Expected: []
    Received: {"activatedUsers": [], "disabledUsers": []}

      981 |       await expect(
      982 |         service.getWorkspaceMembers(workspace.id, owner),
    > 983 |       ).resolves.toStrictEqual([]);
          |                  ^
      984 |
      985 |       expect(workspaceUsers).toHaveBeenCalled();
      986 |     });

      at Object.toStrictEqual (node_modules/expect/build/index.js:174:22)
      at Object.<anonymous> (src/modules/workspaces/workspaces.usecase.spec.ts:983:18)
```